os/arch/arm/src/armv7-m/up_assert.c

```diff
@@ -408,57 +408,12 @@ static void up_dumpstate(void)
 
 static void _up_assert(int errorcode)
 {
-#ifdef CONFIG_BINMGR_RECOVERY
-	int ret;
-	binmgr_request_t request_msg;
-	struct tcb_s *tcb;
-	uint32_t ksram_segment_end = (uint32_t)__ksram_segment_start__ + (uint32_t)__ksram_segment_size__;
-	uint32_t kflash_segment_end = (uint32_t)__kflash_segment_start__ + (uint32_t)__kflash_segment_size__;
-
-	/* Check if the PC lies within kernel context */
-	if ((assert_pc >= (uint32_t)__ksram_segment_start__ && assert_pc <= ksram_segment_end) || (assert_pc >= (uint32_t)__kflash_segment_start__ && assert_pc < kflash_segment_end)) {
-
-		/* Reboot the system if fault is in kernel context */
-
 #ifdef CONFIG_BOARD_ASSERT_AUTORESET
-		boardctl(BOARDIOC_RESET, EXIT_SUCCESS);
+	boardctl(BOARDIOC_RESET, EXIT_SUCCESS);
 #else
-		(void)irqsave();
-		for (;;) {
-#ifdef CONFIG_ARCH_LEDS
-			//board_autoled_on(LED_PANIC);
-			up_mdelay(250);
-			//board_autoled_off(LED_PANIC);
-			up_mdelay(250);
-#endif
-		}
-#endif
-	} else {
-
-		request_msg.cmd = BINMGR_FAULT;
-		request_msg.requester_pid = getpid();
-		if (current_regs) {
-			tcb = sched_self();
-			sched_removereadytorun(tcb);
-#if CONFIG_RR_INTERVAL > 0
-			tcb->timeslice = 0;
-#endif
-			tcb->sched_priority = SCHED_PRIORITY_MIN;
-			sched_addreadytorun(tcb);
-		}
-
-		/* Send a message to fault manager with pid of the faulty task */
-		ret = mq_send(g_binmgr_mq_fd, (const char *)&request_msg, sizeof(binmgr_request_t), BINMGR_FAULT_PRIO);
-		DEBUGASSERT(ret == 0);
-		if (current_regs) {
-			tcb = sched_self();
-			current_regs = tcb->xcp.regs;
-		}
-	}
-#else
-
 #ifndef CONFIG_BOARD_ASSERT_SYSTEM_BLOCK
 	/* Are we in an interrupt handler or the idle task? */
+
```

Comment:
Please remove this extra line.

Reply:
I just follow Nuttx style. Nuttx gives an empty line after comment.

Reply:
Ok, As it was not there previously so I leave comment on it.